style/header.css

```diff
@@ -0,0 +1,97 @@
+.logo,
+.logo img {
+  margin: 3px;
+  display: inline-block;
+  width: 10vw;
+  height: auto;
+  margin-right: 10vw;
+  z-index: 1; }
```

Comment:
why parent and child have the same styles. Also is z-index working or moreover necessary

Reply:
fixed this